modules/installation-special-config-encrypt-disk-tang.adoc

```diff
@@ -46,12 +57,18 @@ $ (cat <<EOM
 }
 EOM
 ) | base64 -w0
```

Comment:
You did a good job separating the input from the output. However, I have a question about how the highlighting is supposed to work. Everything from `(cat <<EOM` to `) | base64 -w0` is actually typed. But the color highlighting shows the text color as being the same as the output. That applies to other `cat <<` examples in the text below as well.

Reply:
Harder to copy/past, but you could add leading `> ` as a common continuation prompt (`$PS2`).  Or have the input highlighted as a shell script instead of a terminal session and remove all the prompts.

Reply:
@jboxman, will you PTAL at this styling?